### PR TITLE
refactor: Migrate from Toasts to Notifications

### LINKED ISSUE #314: Refactor: Replace Toasts with Ant Design Notifications
### Is your feature request related to a problem? Please describe.

Currently, the default way of notifying users is through `Toast` interface, offered by [vue-toastification](https://github.com/Maronato/vue-toastification). The proposal, as discussed on Slack, is to use Ant Design's [Notifications](https://antdv.com/components/notification) instead.

This way, we can get rid of the `vue-toastification` dependency.

### Describe the solution you'd like.

Replace the calls to `toast` throughout the web application, while making sure we stick to the original messages, notification types and duration.

### Describe alternatives you've considered

_No response_

### Additional context.

image

### PR DESCRIPTION
**Pull request by CrystalSage**
## Description
Fixes #314.

The first commit replaces `Toast` invocations with Ant Design `Notifications`. The notification styles and duration are almost exact. A new UX oriented change is that `error` and `warning` notifications now last longer (4 seconds typically) than `success` or `info`  notifications (2 seconds typically) since error messages might take more time to read and digest.

The second commit removes `vue-toastification` dependency from `package.json`.

## Testing
- A reliable way is to head over to `/gcbm/upload` to test the info notification.
- The Run/Status/Download buttons on `/gcbm/run` can also be pressed which should display notifications. See below for expected output.

## Additional Context
`/gcbm/upload`
image

`/gcbm/run`
image

### CODE REVIEW

**Comment by CrystalSage** [comment]
PTAL @YashKandalkar and @sohamsshah.

File: flint.ui/src/components/Cards/CardInfoRun.vue [hunk 1]
```diff
@@ -113,13 +111,17 @@ export default {
       axios
         .get(`${process.env.VUE_APP_REST_API_FLINT_EXAMPLE}/spec`)
         .then((response) => {
-          toast.success(`Specification route has been invoked.`, {
-            timeout: 2000
+          notification['success']({
```
Comment (YashKandalkar) [inline flint.ui/src/components/Cards/CardInfoRun.vue:114]: can you change all of the notification calls to `notification.success({...})` or `notification.error({...})`? 
The square bracket notation isn't necessary as the notification types aren't dynamic.

File: flint.ui/src/components/Cards/CardInfoRun.vue [hunk 2]
```diff
@@ -113,13 +111,17 @@ export default {
...
+            message: 'Specification route has been invoked',
+            duration: 2
```
Comment (YashKandalkar) [inline flint.ui/src/components/Cards/CardInfoRun.vue:116]: Please also change all the durations to 5, I think 2s is too low and keeping all the notifications to the same duration will provide consistency.

**Review by YashKandalkar — APPROVED** [review]
Hey, @CrystalSage Thanks for working on this!
I've left some changes, PTAL.

**Review by YashKandalkar — CHANGES_REQUESTED** [review]
didn't mean to click approve 😬

**Comment by CrystalSage** [comment]
@YashKandalkar Done!

File: flint.ui/src/components/Cards/CardInfoRun.vue [hunk 3]
```diff
@@ -177,19 +191,26 @@ export default {
...
+            message: `${error}`,
+            duration: 5
+          })
           console.log(error)
         })
     }
 
     function apiRoute_nonexistent() {
-      toast.warning('No such route exists!', { timeout: 2000 })
+      notification['warning']({
```
Comment (YashKandalkar) [inline flint.ui/src/components/Cards/CardInfoRun.vue:210]: this one slipped out

**Review by YashKandalkar — COMMENTED** [review]
That was quick @CrystalSage :zap: 
You missed a warning tho, you get a :2nd_place_medal:.

File: flint.ui/src/components/Cards/CardInfoRun.vue (hunk 3 above)
Comment (CrystalSage) [reply flint.ui/src/components/Cards/CardInfoRun.vue]: Oops. Fixed! :)

**Comment by YashKandalkar** [comment]
LGTM :rocket:!
@CrystalSage Thanks for working on this!

**Review by YashKandalkar — APPROVED** [review]

**Pull request merged**